### DISCUSSION TITLE
Concern for running Multiple Angular Micro-frontends in Angular App Shell

### DISCUSSION THREAD

**Discussion by beetz12** [webpack/webpack#16921] [Q&A]
We want to have multiple versions of Angular [micro-frontends](https://en.wikipedia.org/wiki/Microfrontend#:%7E:text=Microfrontend%20is%20a%20front%2Dend,for%20multiple%20front%2Dend%20applications) running within an Angular [app shell](https://angular.io/guide/app-shell#:%7E:text=Application%20shell%20is%20a%20way,automatically%20after%20the%20code%20loads). Is module federation using webpack5 currently the best approach? Looking towards future iterations of Angular, such as when Angular Signals are released, are there alternative methods that we should consider? Additionally, when webpack6 is released, it may not be backwards compatible, so we may need to continue using Webpack5. Would this cause any issues?

For example: The Angular Host app is running Angular 15 with one child Angular micro frontend running Angular 14 and one child Angular micro front end running Angular 13. We will need both Angular 14 and 13 micro frontends to communicate with each other and with the Angular 15 host.

**Comment by alexander-akait** [comment]
Please ask this in `angular` repo or related repos, thank you
